src/components/User.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+const data = require('../data/data.json')
+
+const users = data
+
+User.propTypes = {
+  userData: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      phone: PropTypes.string.isRequired,
+      mail: PropTypes.string,
+      department: PropTypes.string.isRequired,
+    })
+  ),
+  skills: PropTypes.arrayOf(
+    PropTypes.shape({
+      skill1: PropTypes.string.isRequired,
+      skill2: PropTypes.string,
+      skill3: PropTypes.string,
+      skill4: PropTypes.string,
+      skill5: PropTypes.string,
+      skill6: PropTypes.string,
+    })
+  ),
+}
+
+export default function User() {
```

Comment:
In your `UsersPage` you correctly used a `map` render one `User` per user object from your user data. Now you need get the props you are setting:
```suggestion
export default function User({name, email, phone, department, skills}) {
```